site/src/pages/museum/index.astro

```diff
@@ -25,7 +25,7 @@ const carouselEntries = [
 ]
 ---
 
-<Layout title="Home">
+<Layout title="Homes">
```

Comment:
I don't think you intended to change this? (This is the file for /fixable/museum/ which should still be singular Home, or Museum Home if we want; you've already changed the correct place for the collection under exhibit-categories)